### ISSUE TITLE
Cannot build on fresh Angular 16 app

### ISSUE THREAD

**Issue by cg-roling** [Michsior14/ngrx-addons#178]
Reproduction:
https://github.com/cg-roling/store-persist-repro-16

I created a new Angular 16 app with `npx @angular/cli@16 new`, installed `ngrx/store` and `ngrx-addons/persist-state`, set up a very basic reducer, and the app fails to serve with typings errors:

```
Error: node_modules/@ngrx-addons/persist-state/lib/persist-state.feature.module.d.ts:5:42 - error TS2314: Generic type 'PersistStateFeatureModule<T>' requires 1 type argument(s).

5     static ɵfac: i0.ɵɵFactoryDeclaration<PersistStateFeatureModule, never>;
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/@ngrx-addons/persist-state/lib/persist-state.feature.module.d.ts:6:43 - error TS2314: Generic type 'PersistStateFeatureModule<T>' requires 1 type argument(s).

6     static ɵmod: i0.ɵɵNgModuleDeclaration<PersistStateFeatureModule, never, never, never>;
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/@ngrx-addons/persist-state/lib/persist-state.feature.module.d.ts:7:43 - error TS2314: Generic type 'PersistStateFeatureModule<T>' requires 1 type argument(s).

7     static ɵinj: i0.ɵɵInjectorDeclaration<PersistStateFeatureModule>;
```

I've also tried with multiple combinations of Angular version (14/15/16) and persist-state version (1.2, 2.0, 

Strangely, it **does** work on this stackblitz, with Angular 15, and I'm unable to figure out what the difference is:
https://stackblitz.com/edit/angular-kpagc2?file=package.json

**Comment by Michsior14** [comment]
Thanks for the detailed bug report and reproduction steps. I've played around and found an easy fix to skip such issue. Will be released shortly.

**Comment by cg-roling** [comment]
Incredible! Thanks so much!!

**Comment by Michsior14** [comment]
Fixed in [3.0.1](https://github.com/Michsior14/ngrx-addons/releases/tag/persist-state-3.0.1)
